pyproject.toml

```diff
@@ -0,0 +1,47 @@
+[build-system]
+requires = [
+    "cmake>=3.24.0,!=3.30.0",
+    "legate-core==24.06.*",
+    "scikit-build>=0.18.0",
+    "setuptools>=70.0",
+] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit dependencies.yaml and run `rapids-dependency-file-generator`.
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "legateboost"
+version = "0.1"
+authors = [
+    {name = "NVIDIA Corporation"},
+]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Topic :: Database",
```

Comment:
Haven't looked into other items yet, but database seems weird.

Reply:
I was surprised to see it here too, but wasn't confident enough in my understanding of this project to say that it was wrong. This list of classifiers was exactly copied from what's already in `setup.py`:

https://github.com/rapidsai/legate-boost/blob/d4edf2d4c44006228139ec81c753b95e10f921e6/setup.py#L70-L79

Reply:
```suggestion
```

Let's remove it.